### PR TITLE
Clean up node descriptions

### PR DESCRIPTION
**Pull request by mutdmour**
- fix typos and grammar in descriptions
- standardize format

### CODE REVIEW

File: packages/nodes-base/nodes/Ftp.node.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export class Ftp implements INodeType {
 		group: ['input'],
 		version: 1,
 		subtitle: '={{$parameter["protocol"] + ": " + $parameter["operation"]}}',
-		description: 'Transfers files via FTP or SFTP.',
+		description: 'Transfers files via FTP or SFTP',
```
Comment (ivov) [inline packages/nodes-base/nodes/Ftp.node.ts:48]:
```suggestion
		description: 'Transfer files via FTP or SFTP',
```

File: packages/nodes-base/nodes/HttpRequest.node.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export class HttpRequest implements INodeType {
 		group: ['input'],
 		version: 1,
 		subtitle: '={{$parameter["requestMethod"] + ": " + $parameter["url"]}}',
-		description: 'Makes a HTTP request and returns the received data',
+		description: 'Makes an HTTP request and returns the received data',
```
Comment (ivov) [inline packages/nodes-base/nodes/HttpRequest.node.ts:35]:
```suggestion
		description: 'Make an HTTP request and return the received data',
```

File: packages/nodes-base/nodes/If.node.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ export class If implements INodeType {
 		icon: 'fa:map-signs',
 		group: ['transform'],
 		version: 1,
-		description: 'Splits a stream depending on defined compare operations.',
+		description: 'Splits a stream depending on defined compare operations',
```
Comment (ivov) [inline packages/nodes-base/nodes/If.node.ts:19]:
```suggestion
		description: 'Split a stream based on comparisons',
```

File: packages/nodes-base/nodes/Zoho/ZohoCrm.node.ts [hunk 4]
```diff
@@ -34,7 +34,7 @@ export class ZohoCrm implements INodeType {
 		subtitle: '={{$parameter["operation"] + ": " + $parameter["resource"]}}',
 		group: ['input'],
 		version: 1,
-		description: 'Consume Zoho CRM API.',
+		description: 'Consume Zoho CRM API',
```
Comment (ivov) [inline packages/nodes-base/nodes/Zoho/ZohoCrm.node.ts:37]: CRM as part of the name? Do other CRM nodes have CRM in their names?

**Review by ivov — CHANGES_REQUESTED** [review]
- `Start the workflow when X events occur` vs `Listen to X events` - The less wordy version might be preferable. If we can assume the user knows what a trigger does, which is reasonable to assume, then we do not need to be so explicit in every trigger node description.

- Open for discussion - I believe the `description` of a node should be a brief description of the service implemented in the node, because `Consume X API` or `Listen to X events` are always implied by the type (regular or trigger) of the node, so their usefulness is very limited. Also, mentioning specific operations is probably best avoided in the descriptions.

File: packages/nodes-base/nodes/Zoho/ZohoCrm.node.ts (hunk 4 above)
Comment (mutdmour) [reply packages/nodes-base/nodes/Zoho/ZohoCrm.node.ts:37]: there's agile crm and it has crm in the name

**Comment by mutdmour** [comment]
thanks @ivov for the feedback. decided to simplify the PR and remove parts I was not sure about https://github.com/n8n-io/n8n/pull/1883

**Pull request closed without merging**